tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/SpecWorkflowTool/SpecWorkFlowTool.cs

```diff
@@ -166,7 +200,7 @@ private async Task<GenericResponse> IsSpecReadyToGenerateSDKAsync(string typeSpe
         }
 
 
-        [McpServerTool, Description("This tool runs pipeline to generate SDK for a TypeSpec project. This tool calls IsSpecReadyForSDKGeneration to make sure Spec is ready to generate SDK.")]
+        [McpServerTool, Description("This tool runs pipeline to generate SDK for a TypeSpec project. This tool calls IsSpecReadyForSDKGeneration and IsReleasePlanReadyToGenerateSDKAsync make sure Spec and Release Plan is ready to generate SDK.")]
```

Comment:
You should remove `IsReleasePlanReadyToGenerateSDKAsync ` from this description. It's a private method so agent is not aware of that method.